src/stravalib/client.py

```diff
@@ -67,6 +67,8 @@ def __init__(
         rate_limit_requests: bool = True,
         rate_limiter: limiter.RateLimiter | None = None,
         requests_session: Session | None = None,
+        token_expires: int | None = None,
```

Comment:
token expires and refresh values are populated by the user when they instantiate the client object

we could just use logic that if self. token_refresh exists, then try to autorefresh?

Reply:
Maybe it's easier to store the refresh token itself as a client property, just like the access token? And use a (private) helper method to determine when the refresh token is expired?

Reply:
Sure. I can add that as well! I'll work on this a bit more over the next week. Open to any and all feedback, ideas, and improvements! Especially given you know more about the use case of an app where you are retrieving data from multiple users!